lib/flowViewport/selection/fvpPrefixPathMapper.cpp

```diff
@@ -17,44 +17,50 @@
 #include <flowViewport/selection/fvpPrefixPathMapper.h>
 #include <flowViewport/fvpUtils.h>
 
+#include <ufe/pathString.h>
+
 // Need Pixar namespace for TF_ diagnostics macros.
 PXR_NAMESPACE_USING_DIRECTIVE
 
 namespace FVP_NS_DEF {
 
 PrefixPathMapper::PrefixPathMapper(
-    Ufe::Rtid              rtid,
     const Ufe::Path&       appPathPrefix, 
     const PXR_NS::SdfPath& sceneIndexPathPrefix
-) : _rtid(rtid), _appPathPrefix(appPathPrefix), 
+) : _appPathPrefix(appPathPrefix), 
     _sceneIndexPathPrefix(sceneIndexPathPrefix)
 {}
 
 PrimSelections PrefixPathMapper::UfePathToPrimSelections(const Ufe::Path& appPath) const
 {
-    // We only handle scene items from our assigned run time ID.
```

Comment:
One of the important changes in this pull request.  Restricting the path mapper to only deal with one kind of UFE data model is overly restrictive, because the main point is mapping a Ufe::Path prefix into a Hydra scene index SdfPath prefix, regardless of which data model(s) are involved.  This was making the path mapping overly complex.